# This message contains data on set of agents state:
# - agents positions at current step;
# - agents velocities at current step;
# - agents sizes. 

geometry_msgs/Point32[] pos
geometry_msgs/Point32[] vel
float32[] rad